### PR TITLE
Rn UI develop --> Dashboard Demo example link

### PR DESCRIPTION
**Pull request by subhamtsd**
I have updated latest code update

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +34,7 @@
     "@types/react-native": "~0.63.2",
     "@types/react-native-calendars": "^1.505.0",
     "@types/react-test-renderer": "^16.9.2",
+    "@types/redux": "^3.6.0",
```
Comment (saurshaz) [inline package.json:37]: rm redux, we don't need it

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "react-native": "0.63.4",
     "react-native-calendars": "^1.1247.0",
     "react-native-easy-grid": "0.2.2",
+    "react-native-gesture-handler": "^1.10.0",
```
Comment (saurshaz) [inline package.json:16]: this shall also be not needed? `    "react-native-gesture-handler": "^1.10.0",`

**Review by saurshaz — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (subhamtsd) [reply package.json]: removed

File: package.json (hunk 1 above)
Comment (subhamtsd) [reply package.json:40]: removed

**Review by saurshaz — APPROVED** [review]

**Review by saurshaz — APPROVED** [review]

**Pull request merged**